frontends/main/src/app-pages/ChatPage/ChatPage.tsx

```diff
@@ -9,9 +9,23 @@ import { useFeatureFlagEnabled } from "posthog-js/react"
 
 const CONVERSATION_OPTTIONS = {
```

Comment:
Just noticiced the typo OPTTIONS -> OPTIONS

Reply:
Fixed